src/components/Navbar/Navbar.js

```diff
@@ -7,7 +7,15 @@ import { useStateContext } from "../../context/StateProvider";
 import { Link } from "react-router-dom";
 
 export const Navbar = ({ checkHome }) => {
-  const { setShowMenu, cart, wishlist } = useStateContext();
+  const {
+    setShowMenu,
+    cart,
+    wishlist,
+    categories,
+    filterDispatch,
+    productsList,
+    productsDispatch,
+  } = useStateContext();
```

Comment:
suggestion - you may name `useStateContext` something that gives more idea about it while reading the code.

Reply:
The reason ive kept it as ```useStateContext``` is because I'm using only one context to handle all the states, excluding auth. I will shift and make the necessary changes in the future and have other context providers but for now I have only one general provider. Thank you for reviewing.